homeassistant/components/xiaomi_aqara/translations/ca.json

```diff
@@ -8,8 +8,7 @@
         "error": {
             "discovery_error": "No s'ha pogut descobrir cap passarel\u00b7la Xiaomi Aqara, prova d'utilitzar la IP del dispositiu que executa Home Assistant com a interf\u00edcie",
             "invalid_interface": "Interf\u00edcie de xarxa no v\u00e0lida",
-            "invalid_key": "Clau de la passarel\u00b7la no v\u00e0lida",
-            "not_found_error": "No s'ha pogut descobrir cap passarel\u00b7la Zeroconf per obtenir informaci\u00f3 necess\u00e0ria, prova d'utilitzar la IP del dispositiu que executa Home Assistant com a interf\u00edcie"
+            "invalid_key": "Clau de la passarel\u00b7la no v\u00e0lida"
```

Comment:
No, I just removed the "not_found_error" everywhere because that is no longer in the code.
I did that manually.
Schould I revert all the changes to the translations files and leave that for after this PR gets merged?
(I am actually not sure what the protecol for translations is....)

Reply:
I think it's enough to run the translations script in develop mode and then just commit those changes, after modifying `strings.json`. Don't change any translations manually.

Reply:
Thanks, I ran `python3 -m script.translations develop` it made some minor changes to en.json and I pushed those changes in the latest commit.

Reply:
Please revert all the manual translation changes too.

Reply:
I will do that now

Reply:
done